.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP__API_URL = http://localhost:8081
+REACT_APP_PAYMENT_TYPES=[{"type":"MSISDN","name":"Mobile Number"},{"type":"ACCOUNT_NO","name":"Account Number"},{"type":"EMAIL","name":"Email"},{"type":"PERSONAL_ID","name":"Personal ID"},{"type":"BUSINESS","name":"Business"},{"type":"DEVICE","name":"Device"},{"type":"ACCOUNT_ID","name":"Account ID"},{"type":"IBAN","name":"IBAN"},{"type":"ALIAS","name":"Alias"}]
+REACT_APP_VM_IP=3.254.52.179
+REACT_APP_CLIENT_ID='XaOVhjFTX_H8UiZf-O1TuV4ChixshdO8RqghtA_cRUM'
+REACT_APP_ESIGNET_BASE_URL=esignet.collab.mosip.net
```

Comment:
How do we use these env vars? is it for testing, local development or production?

Reply:
They are used in the code. I just removed them from where they are hard coded and put them in the env file.

Reply:
They are for production use

Reply:
The issue I see, at least, with these values being added to main branch of source code:
```
REACT_APP_VM_IP=3.254.52.179
REACT_APP_CLIENT_ID='XaOVhjFTX_H8UiZf-O1TuV4ChixshdO8RqghtA_cRUM'
```
There's no more that VM, and thus, no that IP. Also I'm not sure it's a good idea to commit client ID to git

Reply:
I agree. Although they were already committed but maybe what needs to be done is commit it in a .env.example

Reply:
Let me address the concern